tests/failpoints_cases/test_stale_read.rs

```diff
@@ -97,30 +99,63 @@ fn stale_read_during_splitting(right_derive: bool) {
         .call_command_on_node(leader2.get_store_id(), request, Duration::from_secs(5))
         .unwrap();
 
-    // LocalRead.
-    let read_quorum = false;
-    must_not_eq_on_key(
+    must_not_stale_read(
         &mut cluster,
         stale_key,
-        v3,
-        read_quorum,
         &region1,
         &leader1,
         &region2,
         &leader2,
+        apply_split,
+    );
+}
+
+fn must_not_stale_read(
+    cluster: &mut Cluster<NodeCluster>,
```

Comment:
This test doesn't apply to `ServerCluster`? Can it be made generic?

Reply:
I think `NodeCluster` is enough, the difference between them is the transport layer, `NodeCluster` uses channel, ServerCluster uses gRPC.